webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/WebhookForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import * as Yup from 'yup';
+
+import { translate as __ } from 'foremanReact/common/I18n';
+import ForemanForm from 'foremanReact/components/common/forms/ForemanForm';
+
+import WebhookFormTabs from './Components/WebhookFormTabs';
+
+import { HTTP_METHODS } from './constants';
+
+import './WebhookForm.css';
+
+const webhookFormSchema = Yup.object().shape({
+  name: Yup.string().required(__('is required')),
+  target_url: Yup.string().required(__('is required')),
```

Comment:
I've removed `.url` validator because of: https://github.com/jquense/yup/issues/224. If someone know something like `URI.regexp` in javascript, I'll add it here. For know the URL is being validated on the server.

Reply:
Actually, it will be better to leave it as it is, because any regex could not allow ERB tags in URL (which is supported by this plugin, or at least will be). Server-side validation only is OK for now.